src/runtime_src/doc/toc/xbflash2.rst

```diff
@@ -11,9 +11,15 @@ The Xilinx (R) Board Flash utility (xbflash2) is a standalone command line utili
 
 In 2022.1 release, this utility is Early Access with limited validation.
 
-This tool supported for all Alveo platforms, U2 and U30 platforms.
+This tool supported for all Alveo platforms.
 
-This tool doesn't come with XRT package, it comes as separate xbflash package.
+This tool doesn't require XRT Package and it doesn't come with XRT package, it comes as separate xbflash package.
```

Comment:
It does not. It comes in a `xrt-xbflash2` package:
```
i   xrt-xbflash2                                       - Xilinx RunTime stack for use with Xilinx FPGA platforms     
```
Note that the package description is rather wrong because it copies the one from XRT.